pyconpune/staticfiles/css/main.css

```diff
@@ -7,15 +7,15 @@
     padding-top: 50px; /* margin-top = header's height */
```

Comment:
The indentation changes still have many indentation errors. In case you are changing the indentation then make it same all through out. Some places there are 2 spaces, some place it is 4 and some place 8. Please re-check.